docs/outlook/limits-for-activation-and-javascript-api-for-outlook-add-ins.md

```diff
@@ -70,17 +70,17 @@ Follow these guidelines when designing activation rules for [contextual Outlook
 |:-----|:-----|
 |Clients use the C++ regular expression engine provided as part of the Visual Studio standard template library. This engine complies with ECMAScript 5 standards. |Clients use regular expression evaluation that's part of JavaScript. It's provided by the browser and supports a superset of ECMAScript 5.|
-|Because of the different regex engines, a regex that includes a custom character class based on predefined character classes may return different results in Outlook on Windows and on Mac versus Outlook on the web and on mobile devices.<br/><br/>As an example, the regex `[\s\S]{0,100}` matches any number, between 0 and 100, of single characters that is a whitespace or a non-whitespace. This regex returns different results depending on your Outlook client.<br/><br/>As a workaround, you should rewrite the regex as `(\s\|\S){0,100}`. This regex matches any number, between 0 and 100, of white space or non-white space.<br/><br/>You should test each regex thoroughly on each Outlook client. If a regex returns different results, rewrite the regex. |You should test each regex thoroughly on each Outlook client. If a regex returns different results, rewrite the regex.|
-|By default, the evaluation of all regular expressions for an add-in is limited to one second. Exceeding this limit causes reevaluation of up to three times. Beyond the reevaluation limit, Outlook on Windows and on Mac prevent the add-in from running for the same mailbox in any of the Outlook clients.<br/><br/>Administrators can override these evaluation limits by using the `OutlookActivationAlertThreshold` and `OutlookActivationManagerRetryLimit` registry keys.|The resource monitoring and registry settings supported in Outlook on Windows and on Mac aren't supported in Outlook on the web and on mobile devices. However, add-ins with regular expressions that require excessive amount of evaluation time on Outlook on Windows and on Mac are disabled for the same mailbox on all the Outlook clients.|
+|Because of the different regex engines, a regex that includes a custom character class based on predefined character classes may return different results in Outlook on Windows and on Mac versus Outlook on the web, on mobile devices, and on [new Outlook on Windows (preview)](https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627).<br/><br/>As an example, the regex `[\s\S]{0,100}` matches any number, between 0 and 100, of single characters that is a whitespace or a non-whitespace. This regex returns different results depending on your Outlook client.<br/><br/>As a workaround, you should rewrite the regex as `(\s\|\S){0,100}`. This regex matches any number, between 0 and 100, of white space or non-white space.<br/><br/>You should test each regex thoroughly on each Outlook client. If a regex returns different results, rewrite the regex. |You should test each regex thoroughly on each Outlook client. If a regex returns different results, rewrite the regex.|
```

Comment:
```suggestion
|Because of the different regex engines, a regex that includes a custom character class based on predefined character classes may return different results in Outlook on Windows and on Mac versus in Outlook on the web, on mobile devices, and in [new Outlook on Windows (preview)](https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627).<br/><br/>As an example, the regex `[\s\S]{0,100}` matches any number, between 0 and 100, of single characters that is a whitespace or a non-whitespace. This regex returns different results depending on your Outlook client.<br/><br/>As a workaround, you should rewrite the regex as `(\s\|\S){0,100}`. This regex matches any number, between 0 and 100, of white space or non-white space.<br/><br/>You should test each regex thoroughly on each Outlook client. If a regex returns different results, rewrite the regex. |You should test each regex thoroughly on each Outlook client. If a regex returns different results, rewrite the regex.|
```